### PR TITLE
Cloud Analytics Dashboard Page basic UI layout with required CSS styles.

### PR DESCRIPTION
**Pull request by divakarkonakalla**
@hrushikesh07 , Please merge the changes.

### CODE REVIEW

**Comment by hrushikesh07** [comment]
This branch has conflicts that must be resolved

**Comment by divakarkonakalla** [comment]
Resolved all the conflicts with base branch.

@hrushikesh07 , Please merge the changes.

File: client/cat3/src/partials/sections/dashboard/analytics/organization/organization.html [hunk 1]
```diff
@@ -1,126 +1,165 @@
...
+    <img src="../cat3/images/Catalyst.png" style="position: absolute;right: 2px;display: none" />
+    <!-- <img src="../cat3/images/org-pie-chart.png" style="margin: 0 0 0 12px;" />
+    <img src="../cat3/images/org-bar-chart.png" style="margin: 0 0 0 70px;" />
+    <img src="../cat3/images/org-trends.png" style="margin: 20px 0 0 150px;" /> -->
+    <!-- <img src="../cat3/images/org-pie-chart-usage.png" style="margin: 0 0 0 12px;" />
+    <img src="../cat3/images/org-bar-chart-usage.png" style="margin: 0 0 0 70px;" />
+    <img src="../cat3/images/org-trends-usage.png" style="margin: 20px 0 0 150px;" /> -->
+</div>
+
+<style>
```
Comment (hrushikesh07) [inline client/cat3/src/partials/sections/dashboard/analytics/organization/organization.html]: Remove the css form html page

**Comment by divakarkonakalla** [comment]
Moved all the CSS styles to css file, removed all the commented code.

@hrushikesh07 , Please merge the changes.

File: client/cat3/src/partials/sections/dashboard/analytics/organization/organization.css [hunk 2]
```diff
@@ -1,281 +1,23 @@
-@charset "UTF-8";
```
Comment (hrushikesh07) [inline client/cat3/src/partials/sections/dashboard/analytics/organization/organization.css]: Create scss file & delete organization.css.

File: client/cat3/src/partials/sections/dashboard/analytics/organization/organization.css [hunk 3]
```diff
@@ -1,281 +1,23 @@
...
 }
 
-.widget-fa {
-  font-size: 60px;
-  color: #fff;
+.no-padding {
+  padding: 0;
 }
 
 .active {
```
Comment (hrushikesh07) [inline client/cat3/src/partials/sections/dashboard/analytics/organization/organization.css]: Use parent ID .

**Comment by divakarkonakalla** [comment]
review comments has been taken care

@hrushikesh07 , Please merge the changes.

**Pull request merged**